get_github_coderabbit_review/coderabbit_fetcher/patterns/factory.py

```
@@ -0,0 +1,343 @@
+            )
+
+        processor_class = self._processor_registry[component_type]
+
+        try:
+            if component_type == 'thread':
+                resolved_marker = config.get('resolved_marker', '🔒 CODERABBIT_RESOLVED 🔒')
+                return processor_class(resolved_marker)
+            else:
+                return processor_class()
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)